imports/plugins/included/default-theme/client/styles/homepage.less

```diff
@@ -0,0 +1,157 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:100,300i,400,700'); 
+ 
+.fashionStatement { 
+  background-image: url('/resources/images/homepage/img1.webp'); 
+  background-size: cover;  
+  background-position: 50px auto;  
+  background-attachment: fixed; 
+  width: 100%; 
+  height:650px;  
+  min-height:650px;  
+  max-height:650px; 
+  overflow: hidden; 
+  position: relative; 
+} 
+ 
+.overlay-image{ 
+  background-color: rgba(19, 13, 13, 0.562); 
+  display: block; 
+  width: 100%; 
+  height: 100%; 
+  position: absolute; 
+  z-index: 999; 
+} 
+ 
+.fashionStatement .statement{ 
+  display: block; 
+  margin: 200px auto 0; 
+  position: absolute; 
+  z-index: 1000; 
+  width: 100%; 
+} 
+
+.fashionStatement .statement .shopping { 
+  background: white; 
+  padding-top: 10px; 
+  padding-bottom: 10px; 
+  border-radius: 10px; 
+  -webkit-transition: color 1s, background 1s; 
+  transition: color 1s, background 1s; 
+} 
+ 
+.featured { 
+  margin-bottom: 50px; 
+  margin-top: 80px; 
+} 
+ 
+#categories { 
+  margin-top: 40px; 
+  display: block; 
+  padding: 0 0 30px; 
+  overflow: hidden; 
+} 
+ 
+.align-categories { 
+  width: 90%; 
+  margin: 10px auto 0; 
+}
+
+.cat { 
+  display: -webkit-box; 
+  display: -ms-flexbox; 
+  display: flex; 
+  -webkit-box-pack: center; 
+  -ms-flex-pack: center; 
+  justify-content: center; 
+  -ms-flex-wrap: wrap; 
+  flex-wrap: wrap; 
+  margin-bottom: 80px; 
+} 
+   
+   
+.line { 
+  border-top: 2px solid rgba(0, 0, 0, 0.356); 
+  width: 200px; 
+  margin: 45px; 
+} 
+   
+.category-div { 
+  display: block; 
+  cursor: pointer; 
+  position: relative; 
+  width: 90%; 
+  border-radius: 1px; 
+  height: 230px; 
+  min-height: 230px; 
+  max-height: 230px; 
+  margin: 0 auto 40px; 
+  overflow: hidden; 
+}
+
+.category-overlay { 
+  display: block; 
+  position: absolute; 
+  height: 100%; 
+  width: 100%; 
+  background: black; 
+  opacity: 0.2 !important; 
```

Comment:
We will not be using `!important` to style any element in this project. Find another way to make this style work.